vault/logical_system_paths.go

```diff
@@ -895,10 +1272,19 @@ func (b *SystemBackend) policyPaths() []*framework.Path {
 				},
 			},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   b.handlePoliciesRead(PolicyTypeACL),
-				logical.UpdateOperation: b.handlePoliciesSet(PolicyTypeACL),
-				logical.DeleteOperation: b.handlePoliciesDelete(PolicyTypeACL),
+			Operations: map[logical.Operation]framework.OperationHandler{
```

Comment:
L1203, L1253, 1301, 1318 could all roll over to `Operations` I suppose.

Reply:
And more in the same file.

Reply:
Same as previous comment.